app/models/message.rb

```diff
@@ -103,12 +103,20 @@ def any_objects_with_requested_signature?
 
   # TODO remove UPVS, FS stuff from core domain
   def form
-    return ::Fs::Form.find(metadata['fs_form_id']) if metadata['fs_form_id'].present?
+    if metadata['fs_form_id'].present?
+      ::Fs::Form.find(metadata['fs_form_id'])
+    elsif all_metadata['posp_id'].present? && all_metadata['posp_version'].present?
+      ::Upvs::Form.find_by(
+        identifier: all_metadata['posp_id'],
+        version: all_metadata['posp_version']
+      )
+    else
+      # TODO forms without posp_id, posp_versions
+      ::Upvs::Form.find_by(
+        identifier: all_metadata['message_type']
+      )
```

Comment:
Implementacia je funkcna kompletne - HTML vizualizacia, PDF vizualizacie, XSD validacie a vsetko ostatne. ALE s formularom to je iba docasny hotfix, nie stabilne riesenie.
Tieto ORSR formulare nie su v MEF a celkovo su o dost ine ako bezne UPVS formulare. Z beznych UPVS formularov vieme z XML obsahu podania jednoducho vyparsovat `posp_id`, `posp_version` a na zaklade toho jednoznacne vieme identifikovat o aky formular sa jedna a dotiahnut spravne XSD, XSLT.
Pri tychto ORSR to tak nie je, takze hotfix je: 
- XSD, XSLT hodime teraz jednorazovo do DB,
- `Upvs::Form` a teda jeho XSD, XSLT budeme s draftom matchovat iba na zaklade `message_type` -> co znamena, ze neberieme vobec do uvahy verziu formulara. 

Myslim, ze docasne takto prezijeme a potom doladime, nakolko treba vyriesit:
- Ako identifikovat z XML obsahu o aky typ formulara sa jedna (s know how by mohol pomoct autogram kod),
- Odkial dotahovat XSD, XSLT - chceli by sme to idealne vsetko tahat z nasho S3 uloziska.